index.js

```diff
@@ -102,9 +102,9 @@ module.exports = function (content) {
     dest: '',
     writeFiles: false,
     formatOptions: config.formatOptions || {
-        ttf: {
-            ts: 0 // it's important to set timestamp to fixed value for ttf generation to have consistent output in long term with the same hash
-        }
+      ttf: {
```

Comment:
I agree with you but these settings are sent to the [`webfonts-generator`](https://github.com/sunflowerdeath/webfonts-generator) package so this default value should maybe be set there instead of here.